lib/Watcher.js

```diff
@@ -68,6 +68,21 @@ export default class Watcher {
 
         const added = [];
         const removed = [];
+
+        // TODO: implement this some other way
+        // findAllFiles(this.workingDirectory, this.excludes)
+        //   .then((files: Array<Object>) => {
+        //     files.forEach(({ path, mtime }: Object) => {
+        //       // add files which contain the following:
+        //       // export * from 'file'
+        //       if (!path.includes('node_modules')
+        //         && fs.readFileSync(path, 'utf8').match(/export\s+\*\s+from/) !== null
+        //       ) {
+        //         added.push({ path, mtime, exportsAll: true });
+        //       }
+        //     });
+        //   });
+
```

Comment:
This commented-out code isn't adding anything at the moment. I suggest opening a separate issue explaining what change in behavior you'd like to see.

Reply:
done